server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -647,7 +647,6 @@ TabletGoalState getGoalState(TabletLocationState tls, MergeInfo mergeInfo) {
             case COMPLETE:
               break;
             case STARTED:
-            case SPLITTING:
```

Comment:
When splitting used to be done we needed to host tablets in order to split them.   Now that splitting is no longer needed tablets need to be unhosted.  Wondering if the merge code will host a tablet and then unhost here.  Not sure how the larger code around these states works.

Reply:
Opened #3780